attributes/repo.rb

```diff
@@ -42,6 +42,7 @@
 default['ros_buildfarm']['repo']['pulp_worker_count'] = 2
 default['ros_buildfarm']['repo']['enable_pulp_services'] = true
 default['ros_buildfarm']['rpm_repos']['rhel']['8'] = %w[x86_64]
+default['ros_buildfarm']['rpm_bootstrap_url'] = 'http://repos.ros.org/repos/$distname/ros_bootstrap/$releasever/$basearch/'
```

Comment:
```suggestion
default['ros_buildfarm']['rpm_bootstrap_urls'] = ['http://repos.ros.org/repos/$distname/ros_bootstrap/$releasever/$basearch/']
```

Propose making this plural to match the content generated by this attribute.